From: Benda Xu <heroxbd@gentoo.org>
Date: Thu, 11 Sep 2014 18:20:44 +0900
Subject: Disable class and collaboration graph generation

The graphviz-generated class and collaboration graph
occupies 1GB disk space.  It is too big to deal with.  We turn them
off only to save space.  Output image format is specified as svg to
further save space (bug #751947).
---
 doxygen.cfg | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doxygen.cfg b/doxygen.cfg
index 9071e88..94b7ccb 100644
--- a/doxygen.cfg
+++ b/doxygen.cfg
@@ -2103,7 +2103,7 @@ DOT_FONTPATH           =
 # The default value is: YES.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-CLASS_GRAPH            = YES
+CLASS_GRAPH            = NO
 
 # If the COLLABORATION_GRAPH tag is set to YES then doxygen will generate a
 # graph for each documented class showing the direct and indirect implementation
@@ -2112,7 +2112,7 @@ CLASS_GRAPH            = YES
 # The default value is: YES.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-COLLABORATION_GRAPH    = YES
+COLLABORATION_GRAPH    = NO
 
 # If the GROUP_GRAPHS tag is set to YES then doxygen will generate a graph for
 # groups, showing the direct groups dependencies.
@@ -2215,7 +2215,7 @@ DIRECTORY_GRAPH        = NO
 # The default value is: png.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-DOT_IMAGE_FORMAT       = png
+DOT_IMAGE_FORMAT       = svg
 
 # If DOT_IMAGE_FORMAT is set to svg, then this option can be set to YES to
 # enable generation of interactive SVG images that allow zooming and panning.
